R/zchunk_LB163.bio_Yield_R_GLU_irr.R

```diff
@@ -39,51 +41,160 @@ module_aglu_LB163.bio_Yield_R_GLU_irr_DISABLED <- function(command, ...) {
     L151.ag_rfdHA_ha_ctry_crop <- get_data(all_data, "L151.ag_rfdHA_ha_ctry_crop")
     L151.ag_rfdProd_t_ctry_crop <- get_data(all_data, "L151.ag_rfdProd_t_ctry_crop")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
+
+    # Perform computations
+    # old comment: This method follows the same method as LB113, with the exception that
+    #              the yield indices are computed separately for rainfed/irrigated, but
+    #              again against the global average for each crop, across both irrigated
+    #              and rainfed. This method should roughly preserve the global average
+    #              bioenergy yields; what we want to avoid here is increasing the global
+    #              average yields just by separating irr/rfd.
```

Comment:
I know I approved this, but after a conversation with Marshall I want to make a small edit to this comment. Please remove the sentence that starts "This method". This isn't guaranteed to be true as it depends on the ghost share and it isn't necessary.